.github/workflows/pr.yml

```diff
@@ -60,3 +60,38 @@ jobs:
       - name: verify contracts started onchain
         run: |
           curl http://localhost:1317/agoric/vstorage/data/published.agoricNames.instance | grep offerUp
+      
+      - name: Install linux deps
+        run: |
+          sudo apt-get install --no-install-recommends -y \
+          fluxbox \
+          xvfb
+
+      - name: Run xvfb and fluxbox
```

Comment:
this looks like quite a bit of boilerplate to be copying between repos. consider creating a [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows)